.github/workflows/ci.yml

```diff
@@ -60,13 +60,30 @@ jobs:
         with:
           key: ${{ github.job }}
 
-      - name: Run Cargo Tests
-        uses: actions-rs/cargo@844f36862e911db73fe0815f00a4a2602c279505 # pin@v1
+      - name: Run unit tests
+        run: |
+          make unit-test
+
+  rebalance-integration-test:
+    name: Rebalance integration test
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
Do we need to do `devservices up` again in this separate job?

Reply:
We don't because the makefile recipe has build and reset-kafka as prerequisites.
```makefile
test-rebalance: build reset-kafka
```
So when we do `make test-rebalance` it will automatically do `make build` and `make reset-kafka` for us (which does a devservices up & down)